app/constants.js

```diff
@@ -33,6 +33,7 @@ export const STACK_NAME_VALIDATION_REGEX = '^[-_a-z0-9]+$';
 export const TEMPLATE_NAME_VALIDATION_REGEX = '^[-_a-z0-9]+$';
 export const BROWSER_OS_PLATFORM = navigator.userAgent.indexOf('Windows NT') > -1 ? 'win' : 'lin';
 export const NEW_LINE_BREAKER = BROWSER_OS_PLATFORM === 'win' ? '\r\n' : '\n';
+export const REDIRECT_REASON_TIMEOUT = 'AdminInitTimeout';
```

Comment:
I don't think this needs to be a global constant, as it's only used in one file. it defiantly shouldn't be an angular constant (we're trying to remove those, not to add new ones)

Reply:
see lines 38-40